src/scss/2_layout/container.scss

```diff
@@ -3,6 +3,8 @@
  * Defines the layouts for the container section of the page.
  */
 
+@import "../init";
```

Comment:
don't need this @import declaration, also don't refactor the name of the file, keep it as a partial